openedx/core/djangoapps/content/block_structure/config/__init__.py

```diff
@@ -10,12 +10,42 @@
 from .models import BlockStructureConfiguration
 
 # Switches
+# .. toggle_name: INVALIDATE_CACHE_ON_PUBLISH
+# .. toggle_implementation: WaffleSwitch
+# .. toggle_default: False
+# .. toggle_description: Invalidates block structure cache when changes to courses are published.
```

Comment:
I wonder if we could provide more help as to when I would want to have this off or on?  Was this meant to be temporary, or is this required as an option for different use cases?

Reply:
Thanks. Very helpful.
1. Could you begin with “When enabled...” or something like that?
2. Also, maybe replacing “This toggle comes in handy...” with something like “Enabling this toggle is useful for...”
3. Maybe replace “It is also very useful when...” with “It is also useful to enable this when...”.

Reply:
I really appreciate your patience with those reviews by the way. Sorry about that, but I'm not the best with writing documentation, and your comments are exactly what I needed 😃

Reply:
You are doing a great job. Thanks.